modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -0,0 +1,113 @@
+= Kafka Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install the Connector
+
+From within Studio:
+
+. In Studio, create a Mule Project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the name of the connector in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add >*.
```

Comment:
Is the UI Add >
?
Or is that a typo?

Reply:
In the UI:
![AddModulesToProject](https://user-images.githubusercontent.com/8439645/63456179-32ef4b80-c403-11e9-96d4-6416bf8bd231.png)